### PR TITLE
Fixes cases where the bar animates between states with and without a selection.

### PR DESCRIPTION
**Pull request by bicknellr**
cc @blasten, @frankiefu

### CODE REVIEW

**Comment by frankiefu** [comment]
Verified the changes fix the issue in our app.

File: paper-tabs.html [hunk 1]
```diff
@@ -539,6 +540,9 @@
 
       _tabChanged: function(tab, old) {
         if (!tab) {
+          // remove bar without animation
+          this.$.selectionBar.classList.remove('expand');
```
Comment (notwaldorf) [inline paper-tabs.html]: You can use `this.toggleClass` instead too, if you want.

Comment (notwaldorf) [inline paper-tabs.html]: Also, we prefer comments to be sentence cased: `// Remove bar without animation.`

**Comment by notwaldorf** [comment]
Tests are super upset, but the changes LGTM

**Comment by bicknellr** [comment]
Hmm, same issue w/ Firefox + Selenium that's been going on forever...

File: paper-tabs.html (hunk 1 above)
Comment (bicknellr) [inline paper-tabs.html]: Fixed the comments.

**Comment by bicknellr** [comment]
Assuming the only thing wrong with this next set of tests is our Firefox CI issue, I'm going to merge.

**Pull request merged**